---
layout: default
title: Publications
---

<div class='padded-content'>

	<h1>Publications</h1>

	<h3>2020</h3>

	<ul class='spaced-list'>
		<li>
			<b>Dissecting Cell-to-Cell Variation in Single-Cell RNA-Seq Data</b>.
			<br>Gustafsson J, Robinson JL, Inda-Díaz JS, Björnson E, Jörnsten R, Nielsen J.
			<br><i>(Under Review)</i>
		</li>

		<li>
			<b>An Atlas of Human Metabolism</b>.
			<br>Robinson JL, Kocabaş P, Wang H, Cholley PE, Cook D, Nilsson A, Anton M, Ferreira R, Domenzain I, Billa V, Limeta A, Hedin A, Gustafsson J, Kerkhoven EJ, Svensson T, Palsson BØ, Mardinoglu A, Hansson L, Uhlén M, Nielsen J. <i>Sci Signal</i> (2020), <i>13</i>, eaaz1482.
			<br>[<a href='https://stke.sciencemag.org/content/13/624/eaaz1482'>link</a>] [<a href='/papers/SciSignal_2020_AtlasHumanMetab.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2019</h3>

	<ul class='spaced-list'>
		<li>
			<b>The human secretome – the proteins actively secreted in human cells and tissues</b>.
			<br>Uhlen M, Karlsson MJ, Hober A, Svensson AS, Scheffel J, Kotol D, Zhong W, Tebani A, Vunk H, Edfors F, Sjöstedt E, Mulder J, Mardinoglu A, Berling A, Ekblad S, Dannemeyer M, Kanje S, Rockberg J, Lundqvist M,  Malm M, Volk AL, Nilsson P, Månberg A, Dodig-Crnkovic T, Pin E, Zwahlen M, Oksvold P, von Feilitzen K, Häussler RS, Hong MG, Lindskog C, Ponten F, Katona B, Vuu J, Lindström E, Nielsen J, Robinson JL, Ayoglu B, Mahdessian D, Sullivan D, Thul P, Danielsson F, Stadler C, Lundberg E, Voldborg B, Tegel H, Hober S, Forsström B, Schwenk JM, Fagerberg L, Sivertsson Å. <i>Sci Signal</i> (2019), <i>12</i>, eaaz0274.
			<br>[<a href='https://doi.org/10.1126/scisignal.aaz0274'>link</a>] [<a href='/papers/SciSignal_2019_HumanSecretome.pdf'>PDF</a>]
		</li>

		<li>
			<b>A systematic investigation of the malignant functions and diagnostic potential of the cancer secretome</b>.
			<br>Robinson JL, Feizi A, Uhlén M, and Nielsen J. <i>Cell Rep</i> (2019), <i>26</i>, 2622–2635.
			<br>[<a href='https://doi.org/10.1016/j.celrep.2019.02.025'>link</a>] [<a href='https://www.cell.com/cell-reports/pdf/S2211-1247(19)30196-2.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2018</h3>

	<ul class='spaced-list'>
		<li>
			<b>Targeting CDK2 overcomes melanoma resistance against BRAF and Hsp90 inhibitors</b>.
			<br>Azimi A, Caramuta S, Seashore-Ludlow B, Boström J, Robinson JL, Edfors F, Tuominen R, Kemper K, Krijgsman O, Peeper DS, Nielsen J, Hansson J, Brage SE, Altun M, Uhlén M, and Maddalo G. <i>Mol Syst Biol</i> (2018), <i>14</i>, e7858.
			<br>[<a href='https://doi.org/10.15252/msb.20177858'>link</a>] [<a href='/papers/MSB_2018_CDK2.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2017</h3>

	<ul class='spaced-list'>
		<li>
			<b>Anticancer drug discovery through genome-scale metabolic modeling</b>.
			<br>Robinson JL and Nielsen J. <i>Curr Opin Syst Biol</i> (2017), <i>4</i>, 1-8.
			<br>[<a href='https://doi.org/10.1016/j.coisb.2017.05.007'>link</a>] [<a href='/papers/COSB_2017_AnticancerDrugsGEMs.pdf'>PDF</a>]
		</li>

		<li>
			<b>An integrated network analysis reveals that nitric oxide reductase prevents metabolic cycling of nitric oxide by <i>Pseudomonas aeruginosa</i></b>.
			<br>Robinson JL, Jaslove J, Murawski A, Fazen CH, and Brynildsen MP. <i>Metab Eng</i> (2017), <i>41</i>, 67-81.
			<br>[<a href='https://doi.org/10.1016/j.ymben.2017.03.006'>link</a>] [<a href='/papers/MBE_2017_PAO1.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2016</h3>

	<ul class='spaced-list'>
		<li>
			<b>Integrative analysis of human omics data using biomolecular networks</b>.
			<br>Robinson JL and Nielsen J. <i>Mol BioSyst</i> (2016), <i>12</i>, 2953–2964. *Featured on journal cover.
			<br>[<a href='https://doi.org/10.1039/c6mb00476h'>link</a>] [<a href='/papers/MolBioSyst2016Robinson.pdf'>PDF</a>]
		</li>

		<li>
			<b>Starved <i>Escherichia coli</i> preserve reducing power under nitric oxide stress</b>.
			<br>Gowers GOF, Robinson JL, and Brynildsen MP. <i>Biochem Biophys Res Commun</i> (2016), <i>476</i>, 29–34.
			<br>[<a href='https://doi.org/10.1016/j.bbrc.2016.05.082'>link</a>] [<a href='/papers/BBRC_2016_starvedEcoli.pdf'>PDF</a>]
		</li>

		<li>
			<b>Construction and Experimental Validation of a Quantitative Kinetic Model of Nitric Oxide Stress in Enterohemorrhagic <i>Escherichia coli</i> O157:H7</b>.
			<br>Robinson JL and Brynildsen MP. <i>Bioengineering</i> (2016), <i>3</i>, 9.
			<br>[<a href='https://doi.org/10.3390/bioengineering3010009'>link</a>] [<a href='/papers/Bioeng_2016_EHEC_NOmodel.pdf'>PDF</a>]
		</li>

		<li>
			<b>Discovery and dissection of metabolic oscillations in the microaerobic nitric oxide response network of <i>Escherichia coli</i></b>.
			<br>Robinson JL and Brynildsen MP. <i>Proc Natl Acad Sci U S A</i> (2016), <i>113</i>, E1757–E1766.
			<br>[<a href='https://doi.org/10.1073/pnas.1521354113'>link</a>] [<a href='/papers/PNAS_2016_NOoscill.pdf'>PDF</a>]
		</li>

		<li>
			<b>Ensemble Modeling Enables Quantitative Exploration of Bacterial Nitric Oxide Stress Networks</b>,
			<br>in <i>Stress and Environmental Regulation of Gene Expression and Adaptation in Bacteria</i>. Robinson JL and Brynildsen MP (2016), ed. FJ de Bruijn, John Wiley & Sons, Inc., Hoboken, NJ, USA.
			<br>[<a href='https://doi.org/10.1002/9781119004813.ch98'>link</a>] [<a href='/papers/FJ_de_Bruijn_2016_ch17-6.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2015</h3>

	<ul class='spaced-list'>
		<li>
			<b>An ensemble-guided approach identifies ClpP as a major regulator of transcript levels in nitric oxide-stressed <i>Escherichia coli</i></b>.
			<br>Robinson JL and Brynildsen MP. <i>Metab Eng</i> (2015), <i>31</i>, 22–34.
			<br>[<a href='https://doi.org/10.1016/j.ymben.2015.06.005'>link</a>] [<a href='/papers/MBE_2015_ClpP_nitric_oxide.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2014</h3>

	<ul class='spaced-list'>
		<li>
			<b>Model-Driven Identification of Dosing Regimens that Maximize the Antimicrobial Activity of Nitric Oxide</b>.
			<br>Robinson JL, Miller RV, and Brynildsen MP. <i>Metab Eng Commun</i> (2014), <i>1</i>, 12–18.
			<br>[<a href='https://doi.org/10.1016/j.meteno.2014.08.001'>link</a>] [<a href='/papers/MetEngCommun_2014_Dosing.pdf'>PDF</a>]
		</li>

		<li>
			<b>Deciphering nitric oxide stress in bacteria with quantitative modeling</b>.
			<br>Robinson JL, Adolfsen KJ, and Brynildsen MP. <i>Curr Opin Microbiol</i> (2014), <i>19</i>, 16–24.
			<br>[<a href='https://doi.org/10.1016/j.mib.2014.05.018'>link</a>] [<a href='/papers/CurrOpinMicroReview2013.pdf'>PDF</a>]
		</li>
	</ul>

	<h3>2013 and earlier</h3>

	<ul class='spaced-list'>
		<li>
			<b>A Kinetic Platform to Determine the Fate of Nitric Oxide in <i>Escherichia coli</i></b>.
			<br>Robinson JL and Brynildsen MP. <i>PLoS Comput Biol</i> (2013), <i>9</i>, e1003049.
			<br>[<a href='https://doi.org/10.1371/journal.pcbi.1003049'>link</a>] [<a href='/papers/PLoSCompBiol_2013_kineticNOmodel.pdf'>PDF</a>]
		</li>

		<li>
			<b>Rapid Estimation of Activation Enthalpies for Cytochrome-P450-Mediated Hydroxylations</b>.
			<br>Mayeno AN, Robinson JL, and Reisfeld B. <i>J Comput Chem</i> (2011), 32, 639–657.
			<br>[<a href='https://doi.org/10.1002/jcc.21649'>link</a>] [<a href='/papers/JCC_2010_Mayeno_Robinson_Reisfeld.pdf'>PDF</a>]
		</li>

		<li>
			<b>Predicting Activation Enthalpies of Cytochrome-P450-Mediated Hydrogen Abstractions. 2. Comparison of Semiempirical PM3, SAM1, and AM1 with a Density Functional Theory Method</b>.
			<br>Mayeno AN, Robinson JL, Yang RSH, and Reisfeld B. <i>J Chem Inf Model</i> (2009), <i>49</i>, 1692–1703.
			<br>[<a href='https://doi.org/10.1021/ci8003946'>link</a>] [<a href='/papers/JCIM_2009_Mayeno.pdf'>PDF</a>]
		</li>
	</ul>

</div>
